devtools/protocgen.sh

```diff
@@ -1,38 +1,25 @@
 #!/usr/bin/env bash
-set -o pipefail
 
-protoc_gen_gocosmos() {
-  if ! grep "github.com/gogo/protobuf => github.com/regen-network/protobuf" go.mod &>/dev/null ; then
-    echo -e "\tPlease run this command from somewhere inside the shentu folder."
-    return 1
-  fi
+set -eo pipefail
 
-  go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
-}
+# get protoc executions
+go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
 
-protoc_gen_gocosmos
+# get cosmos sdk from github
+go get github.com/cosmos/cosmos-sdk@v0.45.10 2>/dev/null
```

Comment:
It seems the cosmos sdk version is hardcoded here. Does it need to be changed when we upgrade our cosmos sdk version next time?

Reply:
yes, since not setting it will just download the latest possible version.

Reply:
Is it possible to use `go list -m github.com/cosmos/cosmos-sdk` to get the sdk version specified on go.mod?